remove invalid links / info in csintro courses (#4659)
microsoft/pxt-arcade: docs/courses/csintro1/about/teachers.md

# Teacher Resources

> For any questions please check out the [Educator's Lounge](https://forum.makecode.com/c/educators-lounge/24) on our forum.

Reviewer: Maybe - 

```suggestion
For questions about teaching resources, check out the [Educator's Lounge](https://forum.makecode.com/c/educators-lounge/24) on our forum.
```

Author: Ah that's probably better yeah
